crawler/mobile.js

```diff
@@ -145,22 +145,39 @@ mobile.crawl = function(character, full) {
     const nextRe    = /search\?q=(.+)"> Load older Tweets/;
     const doneRe    = /<div class="noresults"> No results for/;
 
+    // Number of Tweets after which retry with another "until:(lastDay)" query
+    // when the history ends. For some popular characters the history ends early.
+    const retryThreshold = 5000;
```

Comment:
Well.. we have to start somewhere :P
I have a notch that Twitter tries to keep a history of roughly 10k tweets for popular searches. But I actually have no idea. So I just put a random number large enough to not retry every time.